rules-fail2ban.yaml

```diff
@@ -34,3 +34,28 @@ http:
             IP:
                 - "::1"
                 - "127.0.0.1"
+            Rules:
```

Comment:
Wouldn't be better that `Rules` be on the same level as `Whitelist` or `Blacklist` in `Dev` ?

Reply:
Yes i will change that